src/sql/workbench/parts/notebook/node/models/cell.ts

```diff
@@ -594,6 +595,26 @@ export class CellModel implements ICellModel {
 		return endpoint;
 	}
 
+	private getMultilineSource(source: string | string[]): string | string[] {
```

Comment:
So, I'm pretty sure the content manager uses a `demultiline` method to go from string[] -> string. Given changes in this PR, should we skip this step for a minor perf gain?